### PR TITLE
Barangay function

### PR DESCRIPTION
**Pull request by hydrokat**
Good day! I added a barangay functionality.

### CODE REVIEW

File: lib/barangays.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// @flow
+
+import barangays from './data/barangays.json';
+
+export default {
+  all: (Function = () => barangays),
+  find: (Function = (name: String) =>
+  barangays.find(barangay => barangay.name === name))
```
Comment (pcofilada) [inline lib/barangays.js]: Add some indentation here

File: lib/municipalities.js [hunk 2]
```diff
@@ -1,9 +1,38 @@
...
 import municipalities from './data/municipalities.json';
+import barangays from './data/barangays.json';
 
 export default {
-  all: (Function = () => municipalities),
-  find: (Function = (name: String) =>
-    municipalities.find(municipality => municipality.name === name))
+  all: (Function = () => {
+    return municipalities.map(municipality => {
+
```
Comment (pcofilada) [inline lib/municipalities.js]: Remove extra line

File: lib/municipalities.js [hunk 3]
```diff
@@ -1,9 +1,38 @@
...
+      const municipalityBarangays = barangays.filter(
+        barangay => barangay.citymun === municipality.name
+      );
+
+      return {
+        ...municipality,
+        barangays: municipalityBarangays.map(
+          barangay => barangay.name
+        )
+      }
```
Comment (pcofilada) [inline lib/municipalities.js]: Add semicolon

File: lib/municipalities.js [hunk 4]
```diff
@@ -1,9 +1,38 @@
...
+    })
```
Comment (pcofilada) [inline lib/municipalities.js]: Add semicolon

**Review by pcofilada — CHANGES_REQUESTED** [review]
Thanks for your PR and I appreciate it so much. Just fix some of my request and I'll merge this one.

**Comment by hydrokat** [comment]
I pushed the styling requests you asked.

**Review by pcofilada — APPROVED** [review]

**Pull request merged**